manual/core/query_timestamps/README.md

```diff
@@ -0,0 +1,137 @@
+## Query timestamps
```

Comment:
One 'gothca' with Client timestamps that most aren't aware of is that client timestamps are not used by Lightweight transactions.  We should indicate that here.